pallets/lockdown-mode/src/lib.rs

```diff
@@ -35,15 +35,22 @@ pub mod pallet {
 	#[pallet::generate_store(pub(super) trait Store)]
 	pub struct Pallet<T>(_);
 
-	#[derive(Default)]
 	#[pallet::genesis_config]
-	/// Genesis config for lockdown mode pallet
-	pub struct GenesisConfig {}
+	pub struct GenesisConfig {
+		pub activated: bool,
```

Comment:
I think calling it something like `initial_status` would better convey the intent here.

Reply:
Renamed.